Sources/is-access-granted/main.swift

```diff
@@ -0,0 +1,77 @@
+/* Executable that will return a JSON with the granted permissions:
+ *  {
+ * 		"isAccessibilityGranted": boolean
+ * 		"isScreenRecordingGranted": boolean
+ *	}
+ *  
+ */
+
+ import AppKit
+
+func toJson<T>(_ data: T) throws -> String {
+	let json = try JSONSerialization.data(withJSONObject: data)
+	return String(data: json, encoding: .utf8)!
```

Comment:
Better to use `String(decoding: json, as: UTF8.self)`

Reply:
This one should be in utilities.swift

Reply:
I was not able to find a way to use utilities.swift within is-access-granted. I will look more into it but if you have the answer right away can you guide me ?

Reply:
I meant just create a utilities.swift file here.